### ISSUE TITLE
Неверно использован модификатор 

### ISSUE THREAD

**Issue by dariathehuman** [AlDekker/PetCare#6]
https://github.com/AlDekker/PetCare/blob/967c60f0b00d1e0cf8aa5928a2a111a61c04bc0c/app/index.html#L125

В данном случае нужно было сделать так: 

` <button class="banner__button banner__button--down button" data-micromodal-open="site-modal">Подобрать план</button>`

В документации БЭМ с примерами кода: 

https://ru.bem.info/methodology/block-modification/

**Comment by AlDekker** [comment]
Да, досадная ошибка. Не заметил, что не дописал класс button__banner, т.к. явно его подразумевал делая для него модификатор.
